twitchio/user.py

```diff
@@ -48,11 +48,12 @@
 
 class PartialUser:
 
-    __slots__ = "id", "name", "_http", "_cached_rewards"
+    __slots__ = "id", "name", "login", "_http", "_cached_rewards"
 
-    def __init__(self, http: "TwitchHTTP", id: Union[int, str], name: Optional[str]):
+    def __init__(self, http: "TwitchHTTP", id: Union[int, str], name: Optional[str], login: Optional[str]):
         self.id = int(id)
         self.name = name
+        self.login = login
```

Comment:
I actually purposefully ignored the login because name.lower() is almost always equal to the login
(I've seen one edge case with some funky characters), and a good chunk of the api just straight up doesn't return the login key, so it doesn't make sense to implement

Reply:
Makes sense. I was wondering about that. I've removed the login related code.